docs/terminal/atma.md

```diff
@@ -91,20 +91,18 @@ OCR, RAN or BAY will hand aircraft to ATMA no lower than `A110`, unless co-ordin
 ATMA should make use of track-shortening on STARs to allow for an efficient sequence to be applied. Where an aircraft has been taken away from the STAR and are subsequently required to re-join and follow the STAR, the following RTF shall be used:
 
 !!! hint "RTF for re-joining a STAR"
-    **ATMA**: *New Zealand 677, track direct VINCE to re-join the STAR. When ready descend A050*.
+    **ATMA**: *New Zealand 677, track direct VINCE to re-join the STAR. Descend A050*.
```

Comment:
As we're directing them to rejoin the STAR, we need to ensure they have radar terrain separation before they hit VINCE and descend by the profile. Should add radar terrain.